src/extension/netcdf-out/src/main/java/org/geoserver/web/netcdf/NetCDFSettingsContainer.java

```diff
@@ -78,15 +82,29 @@ public void setShuffle(boolean shuffle) {
     /**
      * Whether to copy attributes from the NetCDF/GRIB source to the main output variable.
      */
-    public boolean isCopyAttributes() {
-        return copyAttributes;
+    public boolean isCopyVariableAttributes() {
```

Comment:
Unfortunately you cannot do this because this class is serialised to XML and changing the name of the bean property would break backwards compatibility. If anyone has enabled this setting in an existing configuration file, the changed setting would be lost. The least bad solution in my view is to leave this `copyAttributes` (and consistent accessor names) throughout, and just add your new `copyGlobalAttributes`  (and consistent accessor names). We cannot break backwards compatibility with saved settings.